src/google/cloud/ndb/model.py

```diff
@@ -562,7 +568,10 @@ def _entity_to_ds_entity(entity, set_key=True):
     """
     data = {}
     for cls in type(entity).mro():
-        for prop in cls.__dict__.values():
+        if not hasattr(cls, '_properties'):
+            continue
+
+        for _, prop in cls._properties.items():
```

Comment:
Why did you not use values() here instead of items()?

Reply:
Oh yes... You right! I'll change it

Reply:
Done